src/Select/Select.tsx

```diff
@@ -26,20 +28,20 @@ export interface SelectProps extends BlockProps {
   children: any
 }
 
-export const injectComponent = (component: React.ReactElement<any>, props: InjectValueAndHandler) => {
-  return React.cloneElement(component, {
+export const injectComponent = (component: ReactElement<any>, props: InjectValueAndHandler) => {
+  return cloneElement(component, {
     ...props,
     ...component.props
   })
 }
 
-const getTriggerComponent = (props: InjectValueAndHandler, valueComponent?: React.ReactElement<any>) => {
+const getTriggerComponent = (props: InjectValueAndHandler, valueComponent?: ReactElement<any>) => {
   if (valueComponent) return injectComponent(valueComponent, props)
```

Comment:
nit: what's our policy for single line if statements? Personally I think we should avoid them. @Michiel87 ?